worker/run.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import argparse
+import asyncio
+import logging
+
+from transcribee_proto.api import TaskType
+from transcribee_worker.worker import Worker
+
+logging.basicConfig(level=logging.INFO)
+
+
+async def main():
+    parser = argparse.ArgumentParser(description="Worker")
```

Comment:
```suggestion
    parser = argparse.ArgumentParser(description="the worker doing ML inference for the transcribee open source transcription platform")
```

Reply:
I changed it to `"The worker for the transcribee open source transcription platform"`, since we not (necessarily) only do ml inference in the worker